src/components/CodeBlockTabs.tsx

```diff
@@ -1,158 +1,147 @@
+
+  if (/Android/.test(userAgent)) return 'android'
 
-  return os;
+  if (/Linux/.test(platform)) return 'linux'
+
+  return null
 }
 
-export default function CodeBlockTabs(props: IProps) {
-    const [value, setValue] = React.useState(0);
-
-    const handleChange = (event: React.ChangeEvent<{}>, newValue: number) => {
-        setValue(newValue);
-    };
-
-    useEffect(() => {
-      let os = getOS()
-      if(os === 'windows') {
-        setValue(0)
-      } else if (os === 'linux') {
-        setValue(0)
-      } else if (os === 'macOS') {
-        setValue(1)
-      } 
-
-    }, [])
-
-    function TabPanel(props: TabPanelProps) {
-        const { children, value, index, ...other } = props;
-      
-        return (
-          <div
-            role="tabpanel"
-            hidden={value !== index}
-            id={`simple-tabpanel-${index}`}
-            aria-labelledby={`simple-tab-${index}`}
-            {...other}
-          >
-            {value === index && (
-              <Box style={{ marginTop: '-12px' }}>
-                <Typography component="div">{children}</Typography>
-              </Box>
-            )}
-          </div>
-        );
-      }
-
-      const AntTabs = withStyles({
-        root: {
-          borderBottom: '1px solid #e8e8e8',
-        },
-        indicator: {
-          backgroundColor: '#3f51b5',
-        },
-      })(Tabs);
-
-      interface StyledTabProps {
-        label: string;
-      }
-      
-      const AntTab = withStyles((theme: Theme) =>
-        createStyles({
-          root: {
-            textTransform: 'none',
-            minWidth: 72,
-            backgroundColor: 'transparent',
-            fontWeight: theme.typography.fontWeightRegular,
-            marginRight: theme.spacing(4),
-            fontFamily: [
-              '-apple-system',
-              'BlinkMacSystemFont',
-              '"Segoe UI"',
-              'Roboto',
-              '"Helvetica Neue"',
-              'Arial',
-              'sans-serif',
-              '"Apple Color Emoji"',
-              '"Segoe UI Emoji"',
-              '"Segoe UI Symbol"',
-            ].join(','),
-            '&:hover': {
-              color: '#3f51b5',
-              opacity: 1,
-            },
-            '&$selected': {
-              color: '#3f51b5',
-              fontWeight: theme.typography.fontWeightMedium,
-            },
-            '&:focus': {
-              color: '#3f51b5',
-            },
-          },
-          selected: {},
-        }),
-      )((props: StyledTabProps) => <Tab disableRipple {...props} />);
+export default function CodeBlockTabs(props: Props): ReactElement {
+  const [value, setValue] = React.useState(0)
+
+  const handleChange = (event: React.ChangeEvent<unknown>, newValue: number) => {
+    setValue(newValue)
+  }
+
+  useEffect(() => {
+    const os = getOS()
+
+    if (os === 'windows') {
+      setValue(0)
+    } else if (os === 'linux') {
+      setValue(0)
```

Comment:
I don't really understand what's going on here. Maybe using a variable name more descriptive than `value` would be helpful. Also maybe using string literals instead of using values `0` and `1` would convey better what is the intent.